internal/v1/handler_blueprints.go

```diff
@@ -653,3 +793,64 @@ func (h *Handlers) DeleteBlueprint(ctx echo.Context, blueprintId openapi_types.U
 	}
 	return ctx.NoContent(http.StatusNoContent)
 }
+
+func (h *Handlers) PatchBlueprintsId(ctx echo.Context, id openapi_types.UUID) error {
+	userID, err := h.server.getIdentity(ctx)
+	if err != nil {
+		return err
+	}
+
+	ctx.Logger().Infof("Fetching blueprint %s", id)
```

Comment:
```suggestion
	slog.InfoContext(ctx.Request().Context(), "fetching blueprint", "id", id)
```

Please do not capitalize logs and prefer structured logging. I think echo logger does support structured fields but AFAIR it is not very easy to use, I would just prefer `slog`. Use method `Infoj` if you want to use that.

Reply:
For the record, structured logging via echo logger looks ugly as heck:

```
echo.Infoj("use infoj method instead", map[string]any{"msg": "test"})
```

That is the reason why I suggest using `slog` which is much cleaner.